src/components/themed/TransactionListTop.tsx

```diff
@@ -350,9 +355,12 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
           stakePolicies
         })
       } else if (stakePolicies.length === 1) {
-        const stakePlugin = getPluginFromPolicy(stakePlugins, stakePolicies[0])
+        const [stakePolicy] = stakePolicies
+        const { stakePolicyId } = stakePolicy
+        const stakePlugin = getPluginFromPolicy(stakePlugins, stakePolicy)
         // Transition to next scene immediately
-        if (stakePlugin != null) navigation.push('stakeOverview', { stakePlugin, walletId: wallet.id, stakePolicy: stakePolicies[0] })
+        const stakePosition = this.state.stakePositionMap[stakePolicyId]
+        if (stakePlugin != null) navigation.push('stakeOverview', { stakePlugin, walletId: wallet.id, stakePolicy: stakePolicy, stakePosition })
```

Comment:
Everything written looks good but I think you could go all the way and include the multiple stake policy case as well. There's no additional logic necessary it's just plumbing the position map through and grabbing the selected position from it on the stakeOptions scene